_examples/ceph-source-example.adoc

```diff
@@ -31,21 +31,34 @@ https://github.com/ceph/ceph-ansible[Ceph Ansible playbooks]
 In case you need to configure your own Ceph environment, follow the
 steps below:
 
-1. Clone https://github.com/ceph/ceph-ansible[ceph-ansible] GitHub repo.
-2. Rename `vagrant_variables.yml.sample` to `vagrant_variables.yml`
-   and edit the file by changing to the following snippet:
+1. Download `ceph-nano` in your computer:
 
-   ...
-   # DEFINE THE NUMBER OF VMS TO RUN
-   mon_vms: 1
-   osd_vms: 3
-   mds_vms: 0
-   rgw_vms: 1
-   ...
+  ...
+  curl -L https://github.com/ceph/cn/releases/download/v1.6.1/cn-v1.6.1-b900468-linux-amd64 -o cn && chmod +x cn && mv cn /usr/local/bin
+  ...
 
-3. Rename also `site.yml.sample` to `site.yml`.
-4. Run `vagrant up` inside the repo.
-5. Follow the link:/get-started[getting started] instructions to create an OpenShift project with the Oshinko web interface running.
+2. Create the Ceph cluster:
+
+  ...
+  cn cluster start ceph-source -d /tmp
+  ...
+
+3. After `cn cluster start` finish, you will receive the URL to access your newly created
+Ceph cluster ans your S3 credentials.
+
+4. Access the Ceph Nano Web Console with address `http://<your-ip-address>:5000`. The figure below shows the Web console page:
+pass:[<img src="/assets/ceph-source-example/ceph-source-1.png" class="img-responsive">]
+
+5. We need to create a bucket and add a file so RADAnalytics can access. Let's first create a bucket: Click on the `Create Bucket` button and type `ceph-source` in the `Bucket Name` field:
+pass:[<img src="/assets/ceph-source-example/ceph-source-2.png" class="img-responsive">]
+
+6. Now we have a bucket, and we need to upload a file. Click on the `ceph-source` we just created. The page below is shown:
+pass:[<img src="/assets/ceph-source-example/ceph-source-3.png" class="img-responsive">]
+
+7. Click on `Upload` button and select the `README.md` file in the Spark distribution you have. *Note:* This file is just an example, you can upload whatever text file you want, but expect different results from this tutorial.
```

Comment:
not everyone will have a spark distribution handy, we should point to a file they can download, maybe give a link to the `README.md` in spark's repo?